universal-application-tool-0.0.1/app/auth/IdcsProfileAdapter.java

```diff
@@ -28,6 +28,40 @@ protected String emailAttributeName() {
     return "user_emailid";
   }
 
+  @Override
+  public UatProfileData mergeUatProfile(UatProfile uatProfile, OidcProfile oidcProfile) {
+    String locale = oidcProfile.getAttribute("user_locale", String.class);
```

Comment:
question: do you know what locale format these use? I'm hoping BCP 47, which is what Java Locales use

Reply:
All I can say for sure is that the value that comes back for me is `en`, which is the right BCP 47 answer.